packages/cli/src/util/logs.ts

```diff
@@ -243,25 +243,12 @@ function sanitize(log: any): string {
   );
 }
 
-function colorize(text: string) {
-  if (isError(text)) {
+function colorize(text: string, log: any) {
+  if (log.level === 'error') {
```

Comment:
Given that `log` is `any`, it's hard to know if this will always have the appropriate info. Can we fallback to the old behavior (ie. the `isError` logic) if `log.level` isn't present to be safe? Perhaps adding a note about it would help us know that it's fallback behavior and we can work to get some type-safety in here in the future.

Reply:
I don't think that will be correct behavior as `level` wouldn't always be present by default. If a log is not error or warning, `level` will be undefined; The legacy checks had some incorrect behavior of marking error and warning so if the `level` is supposed to be undefined, going through legacy checks will add back the incorrect highlight.

Reply:
i can actually just type this

Reply:
Ok if it's easy enough for you to type it that'd be awesome. Either way, not worth me blocking you on this if the fallback suggestion doesn't make sense.